javalanglib/src/main/scala/java/lang/Iterable.scala

```diff
@@ -13,7 +13,14 @@
 package java.lang
 
 import java.util.Iterator
+import java.util.function.Consumer
 
 trait Iterable[T] {
   def iterator(): Iterator[T]
+
+  def forEach(action: Consumer[T]): Unit = {
```

Comment:
According to the JavaDoc, the argument should be a `Consumer[_ >: T]` instead of `Consumer[T]`.